spec/requests/api/v2/build_new_course_spec.rb

```diff
@@ -27,6 +27,7 @@
       class: {
         Course: API::V2::SerializableCourse,
         Subject: API::V2::SerializableSubject,
+        PrimarySubject: API::V2::SerializableSubject,
```

Comment:
you wanna add the rest....

secondary etc

Reply:
Not technically needed yet, I don't think? Offhand, could this be part of a larger bit of work to clean up subjects in the specs in general?

Reply:
that was my offhand comment... i take it as no